docs/database/redis.md

```diff
@@ -17,32 +17,32 @@ described on this page.
 <!-- semconv db.redis(full,tag=tech-specific) -->
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`db.redis.database_index`](../attributes-registry/db.md) | int | The index of the database being accessed as used in the [`SELECT` command](https://redis.io/commands/select), provided as an integer. To be used instead of the generic `db.name` attribute. | `0`; `1`; `15` | `Conditionally Required` If other than the default database (`0`). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`db.query.text`](../attributes-registry/db.md) | string | The full syntax of the Redis CLI command. [1] | `HMSET myhash field1 'Hello' field2 'World'` | `Recommended` [2] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`network.peer.address`](../attributes-registry/network.md) | string | Peer address of the database node where the operation was performed. [3] | `10.1.2.80`; `/tmp/my.sock` | `Recommended` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| [`db.collection.namespace`](../attributes-registry/db.md) | string | The index of the database being accessed as used in the [`SELECT` command](https://redis.io/commands/select). [1] | `0`; `1`; `15` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Open question: java instrumentations use index if it's provided in the connection string. But it's dynamic and can be changed with `SELECT index` command. 
It's per-connection option and in theory can be cached and used until changed with `SELECT`

It's possible to get current index from redis via (`client info`)(https://redis.io/docs/latest/commands/client-info/https://redis.io/docs/latest/commands/client-info/) or `client list` commands, but it's a network call and can't be done on per-operation basis.

Given all of these, I think that some instrumentations may try to populate it, but then they have to support detecting and caching `SELECT index` commands.
If they don't it, then capturing index from the connection string seems unreliable.

Reply:
> If they don't it, then capturing index from the connection string seems unreliable.

👍

if we need/want this in the future, it could be a new attribute e.g. `db.connection.default_namespace`